src/domain/platform/admin/templates/AdminTemplatesSection.tsx

```diff
@@ -16,27 +16,31 @@ import {
   TemplateImportCardComponentProps,
   TemplateWithInnovationPack,
 } from './InnovationPacks/ImportTemplatesDialogGalleryStep';
-import TemplateViewDialog from './TemplateViewDialog';
 import { useNotification } from '../../../../core/ui/notifications/useNotification';
 import { UpdateProfileInput } from '../../../../core/apollo/generated/graphql-schema';
 import ConfirmationDialog from '../../../../core/ui/dialogs/ConfirmationDialog';
 import { WhiteboardPreviewImage } from '../../../collaboration/whiteboard/WhiteboardPreviewImages/WhiteboardPreviewImages';
 import { TemplateBase } from '../../../collaboration/templates/CollaborationTemplatesLibrary/TemplateBase';
 import useLoadingState from '../../../shared/utils/useLoadingState';
 import { GraphQLError } from 'graphql';
+import TemplatePreviewDialog, {
+  TemplatePreviewDialogProps,
+} from '../../../template/templatePreviewDialog/TemplatePreviewDialog';
+import { TemplateType } from '../../../collaboration/InnovationPack/InnovationPackProfilePage/InnovationPackProfilePage';
+import { Link } from 'react-router-dom';
 
 /**
  * @deprecated TODO remove
  */
-export interface Template extends TemplateBase { }
+export interface Template extends TemplateBase {}
```

Comment:
are we using `Template` somewhere else? 
if we are not using this here anymore we may want to just use `TemplateBase` and remove it from here